aplus/local_settings.example.py

```diff
@@ -79,3 +79,7 @@
 #        'level': 'DEBUG',
 #    },
 #})
+
+## Branding
+#BRAND_NAME = 'Your brand name'
```

Comment:
Could you move these immediately after `BASE_URL`. I think that could be a better location. If it feels too crowded, then some spacing could be added.

The default values (those written in settings.py) could be mentioned here. How you feel about `=  'Your brand name (default is A+)'` or such?

Reply:
Hmm.. Maybe just after `ADMINS` would be a better location :thinking: . Any case, I think in the first block of variables. would be better than in the end of the file.